### PR TITLE
Promote frog song rupee locations to sometimes hints

### PR DESCRIPTION
**Pull request by fenhl**
I think it makes sense to have these for consistency with the sometimes hint for ZR Frogs in the Rain.

### CODE REVIEW

**Comment by cjohnson57** [comment]
Maybe it'd be better to demote frogs in the rain? (Will defer to racing people since they think about the effects of hints way harder than I ever will)

**Comment by TreZc0** [comment]
This needs to be decided by race mods and will be evaluated there before the PR is considered for merge.

**Comment by fenhl** [comment]
This has been approved by the race mods. ([source](https://discord.com/channels/274180765816848384/438697911753113600/1032069687154065478))

**Review by Cuphat — APPROVED** [review]

**Pull request merged**
